src/main/java/mike/datastructures/Entity.java

```diff
@@ -229,5 +236,19 @@ public Method copyMethod(String name){
 		}
 		return null;
 	}
+	
+	private visibility convertStringToVis(String visType) {
+	    visType = visType.toUpperCase();
+	    switch (visType) {
+	    case "PUBLIC":
+		return visibility.PUBLIC;
+	    case "PRIVATE":
+		return visibility.PRIVATE;
+	    case "PROTECTED":
+		return visibility.PROTECTED;
+	    default:
+		return null;
```

Comment:
Wherever this method ends up being called we should either check for null, or maybe instead just throw an error here? Throwing an error would help the CLI bug mentioned by @SerbianSensation since we can catch it in the CLI.